emulator/src/main/java/com/scalar/client/tool/emulator/ContractManagerEmulator.java

```diff
@@ -134,4 +141,20 @@ private Contract emulateContract(ContractEntry entry) throws Exception {
     m.invoke(contract, this);
     return contract;
   }
+
+  /**
+   * Emulate the certificate holderId and version for the given Contract instance
+   *
+   * @param contract the contract in which the certificate will be emulated
+   */
+  private void emulateCertificateKey(Contract contract) {
+    if (emulatedCertificateKey == null) {
```

Comment:
I'm wondering what would happen if `set-certificate` is not called and getCertificate() is called.
Does it fail ?
If so, maybe it might be good to set the default certificate since real contracts always can execute `getCertificate`.

Reply:
If `set-certificate` is not called, `getCertificate()` will return `null`. 
Indeed, you are right. Should I set a default value for the certificate such as ? 
- `holderId` = `default_holder_id`
- `version` = `1`

Reply:
Yes, let's do that way.

Reply:
The changes are in commit https://github.com/scalar-labs/scalardl-tools/pull/24/commits/bbee3885170199aa4ce93d93929c6e1b9695e806